### PR TITLE
Add travis and run passes

### PR DESCRIPTION
**Pull request by e111077**
added travis, updated package.json and ran format and gen types on repo.

I also had to change the tests for `app-media-video` to check if it supports setting `MediaStream` as a `HTMLVideoElement.srcObject = MediaStream` or `HTMLVideoElement.src = URL.createObjectURL(MediaStream)`

### CODE REVIEW

**Comment by e111077** [comment]
idk why I can't turn the repo on in travis

File: demo/index.html [hunk 1]
```diff
@@ -18,24 +18,24 @@
...
+          npmScript.onerror = function() {
+            // on error try loading it from the bower directory
+            var bowerScript = document.createElement('script');
+            bowerScript.src = '../../webrtc-adapter/release/adapter.js';
+            bowerScript.onload = resolve;
+            document.body.appendChild(bowerScript);
             document.body.appendChild(npmScript);
-          } else {
-            resolve();
           }
```
Comment (bicknellr) [inline demo/index.html]: This looks like it shouldn't have moved.

File: test/app-media-recorder.html [hunk 2]
```diff
@@ -63,6 +59,8 @@
     </test-fixture>
 
     <script>
+      var EVENT_WAIT_TIME_MS = 5000;
```
Comment (bicknellr) [inline test/app-media-recorder.html]: Is this a timeout?

File: demo/index.html (hunk 1 above)
Comment (bicknellr) [reply demo/index.html]: Realized my comment doesn't make much sense without the `?w=1` view... I think the `document.body.appendChild(npmScript);` needs to be outside the `onerror` handler for it to try to load.

**Comment by e111077** [comment]
Adding @keanulee since Russell is OOO this week.

**Pull request merged**